### PR TITLE
CONSOLE-3806: Remove Protractor

### PR DESCRIPTION
**Pull request by rhamilto**

### CODE REVIEW

File: bill-of-materials.json
Comment (rhamilto) [inline bill-of-materials.json:1]: This is an orphaned file from the CoreOS days.

File: e2e.yaml
Comment (rhamilto) [inline e2e.yaml:1]: This is an orphaned file from the early days of Protractor.

**Comment by rhamilto** [comment]
Work is tech debt and has no customer-facing impact.

/label docs-approved
/label px-approved
/label qe-approved

File: frontend/.eslintignore [hunk 1]
```diff
@@ -9,4 +9,3 @@ public/lib
 Godeps
 @types
 dynamic-demo-plugin
-integration-tests
```
Comment (rhamilto) [inline frontend/.eslintignore:12]: This change resulted in the second commit of lint fixes.  I kept those changes separate so as to not pollute the Protractor removal changes.

**Review by jcaianirh — APPROVED** [review]
/lgtm

**Pull request merged**